### DISCUSSION TITLE
Boot Screen/Linux Console/TTY Resolution Absolute Beginner

### DISCUSSION THREAD

**Discussion by HanumanH** [zbm-dev/zfsbootmenu#344] [Questions]
I have read two other post regarding screen resolution in 'Discussions'. The first really prove how ignorant I am. The second one I was able to get a clue that I need to put 'nvim' in front of 'zbm-efi-kcl -e'. It took me a while to figure out that 'downloading from' https://github.com/zbm-dev/zfsbootmenu/blob/master/bin/zbm-efi-kcl actually mean that I need to follow the link to the page and click on 'RAW' and save the page as a text file onto my arch linux 'Downloads' directory. It took me even longer to realize that my whole command line should look like this:  nvim zbm-efi-kcl -e /efi/EFI/zbm/zfsbootmenu.EFI. Wait for it. The worse story is not here. It took me in total two weeks to finally got an arch linux box with zfs encryption using only zfsbootmenu, no bridging from grub or any other boot loader. That was two weeks averaging 5 hours a day reading blog, youtube video, every instruction on zfsbootmenu wiki tons of article on arch linux wiki and not to mention openzfs where I strongly believe they intentional design their document for computer god not mere mortal. It's ok we can all laugh here. I am that ignorant. :-) LOL. Now one to my real question. Where exactly in the nvim editor do I put this line: rd.vconsole.font=ter-132n. I have already try the middle of the file :wq but that did not work. Could I be completely following the wrong approach. Please kindly help this poor soul. I would really appreciate if you could detail the process/instruction for stupid proofing such as myself.

**Comment by ahesford** [comment]
Putting `nvim` in front of `zbm-efi-kcl -e /efi/EFI/zbm/zfsbootmenu.EFI` probably opened the `zbm-efi-kcl` script itself in `nvim`. What you probably want is to run

```sh
EDITOR=nvim zbm-efi-kcl -e /efi/EFI/zbm/zfsbootmenu.EFI
```

At this point, you probably need to re-fetch the script if you've saved some edits to it.

**Comment by HanumanH** [comment]
Thank you for lending a helping hand. I run the command:
`EDITOR=nvim zbm-efi-kcl -e /efi/EFI/zbm/zfsbootmenu.EFI` but all I got is `command not found: zbm -efi-kcl`. I also tried with vim and nano editor and all show the same message. Have I miss understand the instruction?

**Comment by zdykstra** [comment]
```
chmod +x zbm-efi-kcl
EDITOR=nvim ./zbm-efi-kcl -e /efi/EFI/zbm/zfsbootmenu.EFI
```

`zbm-efi-kcl` is a shell script, so you have to execute it. To execute it, you have to set the eXecute bit (`+x`). Programs are run if they're in your `$PATH` (`env | grep PATH`). Since you downloaded the file to (likely) somewhere in your home directory, the program isn't found by default. Prepending the name with `./` lets your shell find it as long as you're in directory it's in.

**Reply by HanumanH** [reply]
Thank you kindly for being very detail with me. You have no idea how much I appreciate this kind of instruction.

**Reply by HanumanH** [reply]
I fetch the file `zbm-efi-kcl` into 'Download'. Working in 'Download' directory. I executed the following: `chmod +x zbm-efi-kcl.txt` than `EDITOR=nvim ./zbm-efi-kcl.txt -e /efi/EFI/zbm/zfsbootmenu.EFI` when the file open I have only one line in the file. The only line in the file that I have is: `zfsbootmenu loglevel=4 nomodeset` so on line two I place: `rd.vconsole.font=ter-132n` after I proceed to `:wq` and reboot machine. My font did not change. Resolution did not change. Again, I really appreciate you repeating the basic for me; thank you. Now, where did I go wrong this time?
